products/opensuse/main.pm

```diff
@@ -700,7 +700,7 @@ sub load_x11tests() {
             loadtest "x11/reboot_lxde";
         }
     }
-    loadtest "x11/glxgears";
+    loadtest "x11/glxgears" unless !is_staging();
```

Comment:
Double negation?
should be either
if !is_staging()
or
unless is_staging()
right?